cmd/querier/main.go

```diff
@@ -91,13 +92,16 @@ func main() {
 	sampleQueryable := querier.NewQueryable(dist, chunkStore, false)
 	metadataQueryable := querier.NewQueryable(dist, chunkStore, true)
 
-	engine := promql.NewEngine(sampleQueryable, nil)
+	maxConcurrent := 20
+	timeout := 2 * time.Minute
```

Comment:
Both of these should be command-line parameters.

Reply:
let's put these in querier config